### PR TITLE
Url request debug description

### PR DESCRIPTION
**Pull request by marijnschilling**
I can't run the tests they seem to be running on macOSX and not iOS, and don't know how to fix it!

### CODE REVIEW

**Comment by 3lvis** [comment]
@marijnschilling rebase with master, tests should be running fine now.

**Comment by 3lvis** [comment]
I'm not sure if what you want to do can be done.

screen shot 2017-04-03 at 16 45 59

File: Sources/URLRequest+Sweetness.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension URLRequest {
+    public var debugDescription: String {
```
Comment (elland) [inline Sources/URLRequest+Sweetness.swift]: You can't override a method from an extension, AFAIK.

Comment (marijnschilling) [reply Sources/URLRequest+Sweetness.swift]: It's not overriding just adding

Comment (marijnschilling) [reply Sources/URLRequest+Sweetness.swift]: I could do it locally...

Comment (elland) [reply Sources/URLRequest+Sweetness.swift]: It is, `debugDescription` is defined by `NSObject`. Anything that inherits from it has it.

**Comment by marijnschilling** [comment]
Oke i renamed it to `debugLog()` now, and it all works! What do you think?

File: Sources/URLRequest+Sweetness.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension URLRequest {
+    public func debugLog() -> String {
+        var description = "\(self)"
+        if let headers = self.allHTTPHeaderFields {
+            description.append("\n\(headers)")
+        }
+        if let httpBody = self.httpBody, let dataString = String(data: httpBody, encoding: String.Encoding.utf8) {
```
Comment (3lvis) [inline Sources/URLRequest+Sweetness.swift]: This `String.Encoding.utf8` can just be `.utf8`

**Comment by 3lvis** [comment]
I think that works, @marijnschilling :)

**Comment by 3lvis** [comment]
Should I remove the `[WIP]` from the PR title?

**Pull request merged**